grakn-graql-shell/src/main/java/ai/grakn/graql/internal/shell/animalia/chordata/mammalia/artiodactyla/hippopotamidae/Hippopotamus.java

```diff
@@ -35,5 +35,6 @@
 
     void submerge();
 
+    @SuppressWarnings("unused")
     boolean isHungryHungry();
```

Comment:
`@API` for sure

Reply:
Wow you right. Horrible mistake on my side 😄